internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -242,12 +246,16 @@ func (c *ClusterManager) processProvisioningClusters() []error {
 	// process each local pending cluster and compare to the underlying ocm cluster
 	for i := range provisioningClusters {
 		provisioningCluster := provisioningClusters[i]
-		glog.V(10).Infof("provisioning cluster ClusterID = %s", provisioningCluster.ClusterID)
-		metrics.UpdateClusterStatusSinceCreatedMetric(provisioningCluster, api.ClusterProvisioning)
-		_, err := c.reconcileClusterStatus(&provisioningCluster)
-		if err != nil {
-			errs = append(errs, errors.Wrapf(err, "failed to reconcile cluster %s status", provisioningCluster.ClusterID))
-			continue
+		if provisioningCluster.ClusterType != api.Enterprise.String() {
+			glog.V(10).Infof("provisioning cluster ClusterID = %s", provisioningCluster.ClusterID)
+			metrics.UpdateClusterStatusSinceCreatedMetric(provisioningCluster, api.ClusterProvisioning)
+			_, err := c.reconcileClusterStatus(&provisioningCluster)
+			if err != nil {
+				errs = append(errs, errors.Wrapf(err, "failed to reconcile cluster %s status", provisioningCluster.ClusterID))
+				continue
+			}
+		} else {
+			glog.V(10).Infof("skipping reconciliation of provisioning %s cluster with ClusterID = %s", api.Enterprise.String(), provisioningCluster.ClusterID)
```

Comment:
Let's take the comment in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1429/files#r1039519780 onto account here